### ISSUE TITLE
Docker: Fetching repo using access tokens.

### ISSUE THREAD

**Issue by Destidom** [autograde/aguis#97]
If a user tries to push content to be tested by autograde, the users access token is used by the system to fetch the repo.
If the current user has no admin rights on the organization it will not be able to see the private repositories, hence will not be able to download the hidden test folder.

**Comment by Destidom** [comment]
Added functionality for fetching test folder through the persons who created the course access token. Thus being able to download the test repo.

A new problem is that it seems like normal user access tokens will not allow the user to download their own repo, needs further investigation.

**Comment by Destidom** [comment]
I looked more into how the old autograde system did this, and there we used an admin token for fetching repositories.
Seems like if we don't let the user create a repo using their own access-token we will not be able to find that repo again, and since autograde is creating the repos and connecting them to a team, the repo has no assosication on the users access-tokens.

So temporary fix until we have something more tangible will be to use admin token.
